package.json

```
@@ -12,7 +12,7 @@
   ],
   "scripts": {
     "build": "ncc build action.js -o dist",
-    "test": "npm run lint && node test.js",
+    "test": "npm run lint && node setup-test.js",
     "lint": "standard"
   },
   "author": "RafaelGSS <rafael.nunu@hotmail.com>",
```

Add [engines](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) field?
```suggestion
  },
  "engines": {
    "node": ">=0.12.0"
  },
  "author": "RafaelGSS <rafael.nunu@hotmail.com>",
```

---

Since we are supporting all versions, there's no point in having it

---

Also, there's a good amount of developers that do not agree with the `engines` field -- I particularly attempt to avoid it unless I'm really using a feature from a specific Node.js version. In 2012, Isaacs tried to remove it https://groups.google.com/g/nodejs/c/_ZggTOJQ2Rk

---

The engines field is used by lot of libraries, an [example search](https://sourcegraph.com/search?q=context:global+path:package.json+%27%22engines%22%27&patternType=keyword&sm=0) here.

But that's okay since the module is supporting all Node.js versions.